corehq/apps/couch_sql_migration/parallel.py

```diff
@@ -165,12 +166,12 @@ def is_stopped(self):
         return self._stop.is_set() or not self.running
 
 
-class _Stop:
-    pass
+class STOP:
+    """Constant/singleton stop token"""
```

Comment:
I've seen the convention `STOP = object()  # Constant/singleton stop token` used before. I think that'd work pretty much the same as this if an all-caps `class` name bugs you

Reply:
That will not work in this case because it would not be (un)pickled across process boundaries as the same object. Classes have the handy property of always being unpickled as the same object. I.E.
```py
import pickle

class ONE:
    pass

TWO = object()

assert pickle.loads(pickle.dumps(ONE)) is ONE, "new ONE"
assert pickle.loads(pickle.dumps(TWO)) is TWO, "new TWO"
# AssertionError: new TWO
```

Reply:
Gotcha, neat